### PR TITLE
Support Directives on Subscriptions

### PR DESCRIPTION
**Pull request by jeffutter**
This supports the same directives that are supported on Object on Subscriptions.

My main use-case here is to support WunderGraph EDFS, which requires directives on Subscription fields: https://cosmo-docs.wundergraph.com/router/event-driven-federated-subscriptions-edfs

### CODE REVIEW

**Comment by xamgore** [comment]
Do you have any idea, why the pipeline checks could fail?

**Comment by jeffutter** [comment]
@xamgore I believe I fixed this up by updating `parser/src/parse/generated.rs`

File: tests/type_directive.rs [hunk 1]
```diff
@@ -50,10 +51,36 @@ pub fn test_type_directive_1() {
...
+
+    let schema = Schema::build(Query, EmptyMutation, Subscription)
+        .enable_subscription_in_federation()
+        .finish();
 
     let sdl = schema.sdl_with_options(SDLExportOptions::new().federation().compose_directive());
 
+    println!("----");
+    println!("{}", sdl);
+
```
Comment (xamgore) [inline tests/type_directive.rs:83]: For me everything else looks fine except these probably forgotten debug statements.

Comment (jeffutter) [reply tests/type_directive.rs]: Fixed. Thanks for catching it.

**Comment by xamgore** [comment]
@sunli829 would you be so kind to pay your attention here, please?

**Comment by sunli829** [comment]
sorry for the late processing, it's been a busy time. 😂

**Pull request merged**